src/ajax/core.cljs

```diff
@@ -15,6 +15,8 @@
   (:require-macros [ajax.macros :as m]
                    [poppea :as p]))
 
+(def default-interceptors (atom ()))
```

Comment:
The benefit of `[{:req fn :res fn}]` is that it will be easier to parse which request / response pairs belong together, and it may simplify managing the order of the chain.

But I can work with either format, so it's your call.